### PR TITLE
Add: benchmark

### PR DESCRIPTION
**Pull request by noob77777**
# Added Benchmarks

## Description
Benchmarks added. JuCC and JuCCCompute
JuCC includes all tasks and file operations
JuCCCompute does not consider file operations

image

## Further work
Add more benchmarks for individual components in the future.

### CODE REVIEW

File: benchmark/input/benchmark_grammar.g [hunk 1]
```diff
@@ -0,0 +1,98 @@
+## This is the grammar file for JuCC
```
Comment (bisakhmondal) [inline benchmark/input/benchmark_grammar.g:1]: do we need this to duplicate multiple times🤔

**Review by bisakhmondal — DISMISSED** [review]
lgtm

**Comment by bisakhmondal** [comment]
How about adding the benchmark to main `readme.md`!!

File: benchmark/input/benchmark_grammar.g (hunk 1 above)
Comment (noob77777) [reply benchmark/input/benchmark_grammar.g:1]: incase we change main grammar with out updating benchmarks

File: README.md [hunk 2]
```diff
@@ -43,6 +43,17 @@ $ ninja
...
+Note: `-DCMAKE_BUILD_TYPE=Release` is needed
+
+```
+$ mkdir build
+$ cd build
+$ cmake -GNinja -DCMAKE_BUILD_TYPE=Release ..
+$ ninja
+$ ./bin/jucc_benchmark
+```
+
```
Comment (bisakhmondal) [inline README.md:56]: along with the stats.

Comment (noob77777) [reply README.md:56]: stats ta toh khub baje :clown_face:

Comment (bisakhmondal) [reply README.md:56]: F

**Review by bisakhmondal — APPROVED** [review]

**Pull request merged**